content/en/basics/how-storage-works/filecoin-plus/index.md

```diff
@@ -130,23 +130,23 @@ Once you have received DataCap to an address, you can check the remaining balanc
 
 #### With lotus v1.10.0 ^
 
-```
+```shell
 lotus filplus check-client-datacap f00000
 ```
 
 #### With lotus v1.9.0 and below
 
 _Note: [Lotus-shed](https://github.com/filecoin-project/lotus/tree/master/cmd/lotus-shed) is a separate package you will need to build and install (`make lotus-shed` in the [Lotus](https://github.com/filecoin-project/lotus) source), although these features are slated to be merged into Lotus._
 
-```
+```shell
 lotus-shed verifreg check-client f00000
 ```
 
 ### Finding storage providers to take Fil+ deals
 
 There are a few different ways in which a client can find a storage provider to take a Fil+ storage deal:
 
-1. In the **For Clients** section of the [Filecoin Plus Registry](https://plus.fil.org/), there is a [Miner Registry](https://plus.fil.org/miners) which lists a self-selected set of storage providers who are willing to take Fil+ storage deals
+1. In the **For Clients** section of the [Filecoin Plus Registry](https://filplus.fil.org/), there is a [Miner Registry](https://filplus.fil.org/#/miners) which lists a self-selected set of storage providers who are willing to take Fil+ storage deals
```

Comment:
Note:

The link detector flagged https://plus.fil.org/miners as 404. When I went to the link, noticed ti redirected to https://filplus.fil.org/. So I updated links. Can someone who knows the program confirm that these are the new links?